socorro/external/boto/crash_data.py

```diff
@@ -24,3 +28,53 @@ def get_storage(self):
         # implementation details with boto S3.
         return self.config.hbase.hbase_class(self.config.hbase)
 
+
+class SimplifiedCrashData(BotoS3CrashStorage):
+    """The difference between this and the base CrashData class is that
+    this one only makes the get() and if it fails it does NOT
+    try to put the crash ID back into the priority jobs queue.
+    Also, it returns a python dict instead of a DotDict which
+    makes this easier to work with from the webapp's model bridge.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(SimplifiedCrashData, self).__init__(*args, **kwargs)
+        # Forcibly set this to override the default in the base
+        # crash storage class for boto. We're confident that at this
+        # leaf point we want to NOT return a DotDict but just a plain
+        # python dict.
+        self.config.json_object_hook = dict
```

Comment:
Hopefully this can be removed once we get rid of the old CrashData classes and stuff.